test/fail_compilation/traits_parameters.d

```diff
@@ -0,0 +1,10 @@
+/*
+TEST_OUTPUT:
+---
+fail_compilation/traits_parameters.d(9): Error: `__traits(parameters)` cannot have arguments, but `234` was supplied
+fail_compilation/traits_parameters.d(10): Error: `__traits(parameters)` may only be used inside a function
+---
+*/
+
+typeof(__traits(parameters, 234)) xyz;
+typeof(__traits(parameters)) x;
```

Comment:
Why this was marked as resolved when clearly is not? Stuff like this can trigger silent bad behaviour when piping a file into a POSIX compliant tool. See problems related to not including a newline in the files: https://stackoverflow.com/questions/72271/no-newline-at-end-of-file-compiler-warning and https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file . C/C++ compilers often workaround this due to editors not adding them by default but if we use something like a `grep`/`cat`/`sed` or a combination of those, the last line can be silently skipped due to this and therefore can break downstream projects.

Reply:
If I resolved it it was by accident. I'll add one when I rebase this